babel.config.locales.js

```diff
@@ -0,0 +1,41 @@
+// Create UTC creation date in the correct format.
+const potCreationDate = new Date()
+  .toISOString()
+  .replace('T', ' ')
+  .replace(/:\d{2}.\d{3}Z/, '+0000');
+
+module.exports = {
+  extends: './babel.config.js',
+  plugins: [
+    [
+      'module:babel-gettext-extractor',
+      {
+        headers: {
+          'Project-Id-Version': 'amo',
+          'Report-Msgid-Bugs-To': 'EMAIL@ADDRESS',
+          'POT-Creation-Date': potCreationDate,
+          'PO-Revision-Date': 'YEAR-MO-DA HO:MI+ZONE',
+          'Last-Translator': 'FULL NAME <EMAIL@ADDRESS>',
+          'Language-Team': 'LANGUAGE <LL@li.org>',
+          'MIME-Version': '1.0',
+          'Content-Type': 'text/plain; charset=utf-8',
+          'Content-Transfer-Encoding': '8bit',
+          'plural-forms': 'nplurals=2; plural=(n!=1);',
```

Comment:
some of this looks a bit boilerplate-y (e.g. `EMAIL@ADDRESS`) - does it need updating?

Reply:
Copied verbatim from [here](https://github.com/mozilla/addons-frontend/blob/master/webpack.l10n.config.babel.js#L24-L55). I could actually remove the webpack config file while we are here as that is no longer referenced or needed.